ScoutSuite/providers/aws/facade/iam.py

```diff
@@ -99,6 +99,7 @@ async def get_users(self):
         await get_and_set_concurrently(
             [functools.partial(self._get_and_set_inline_policies, iam_resource_type='user'),
              self._get_and_set_user_groups,
+             self._get_and_set_user_tags,
```

Comment:
Is this necessary? My understanding is that [list_users already contains the Tags](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.list_users), we shouldn't have to make all these additional calls?

Reply:
I had to call to "list_user_tags" for getting the tags. Without that call, the user don't contain the Tags.

Reply:
I've run a quick test and can't see any tags being returned. I've looked into list_users with boto3 and also with the CLI (list-users) and same results, no tags, even though it's in the documentation as you've mentioned (https://docs.aws.amazon.com/cli/latest/reference/iam/list-users.html).